Library/Homebrew/cmd/install.rb

```diff
@@ -71,7 +71,13 @@ def install
 
         if f.installed?
           msg = "#{f.full_name}-#{f.installed_version} already installed"
-          msg << ", it's just not linked" unless f.linked_keg.symlink? || f.keg_only?
+          unless f.linked_keg.symlink?
+            if Pathname.new("#{HOMEBREW_CELLAR}/#{f.oldname}").exist?
+              msg << ", it's just not migrated"
```

Comment:
No, I don't think so. Because if `f.installed?` returns true, then `HOMEBREW_CELLAR/newname` will definitely exist. In this state, it cannot be pre-migration.